build.sbt

```diff
@@ -22,7 +22,8 @@ initialize := {
   initialize.value
 }
 
-// akka.AkkaBuild.buildSettings
+scalaVersion := Dependencies.scala212
```

Comment:
isn't it confusing to define the scalaVersion both here and in Dependencies.scala?